### ISSUE TITLE
High disk read operations during bulk_stats_calculating_job make the remote instance unresponsive

### ISSUE THREAD

**Issue by sermelipharo** [Freika/dawarich#448]
**Describe the bug**  
The `bulk_stats_calculating_job` causes an abnormally high number of disk read operations, rendering the entire instance unresponsive. This occurs during the job's default schedule every six hours. The issue persists for 1.5–2 hours per execution. The deployment uses the standard `docker-compose.yml` file, not the `docker-compose_mounted_volumes.yml`. Adding I/O throttling labels in the `docker-compose.yml` configuration does not mitigate the problem.  

**Version**  
Versions where the issue was observed: `0.16.x` and `0.17.0`.

**To Reproduce**  
1. Deploy using the standard `docker-compose.yml` file.  
2. Wait for the `bulk_stats_calculating_job` to execute on its default schedule.  
3. Observe the system's responsiveness during the job execution.  

**Expected behavior**  
The job should execute without making the remote instance unresponsive, even with a large dataset.

**Screenshots**  
<details><summary><b>Expand</b></summary><img width="975" alt="image" src="https://github.com/user-attachments/assets/964723c2-7a04-459c-991d-d316c7449e8e"></details>

**Additional context**  
- Host: Vultr standard cloud compute instance (2 vCPUs, 4096 MB RAM).  
- I've tried to add labels into `docker-compose.yml` to limit I/O operations:  
  ```yaml
  labels:
    io.weight: "500"
    io.iops.read: "/dev/sda:500"
    io.iops.write: "/dev/sda:500"
  ```  
  But this settings had no effect on the issue.  
- Current dataset statistics: 3,000,000+ geopoints tracked and ≈747,000 km distance, likely contributing to the high resource usage.

**Comment by sermelipharo** [comment]
Also to notice: there were no major updates between calc jobs, import has been finished weeks ago

**Comment by Freika** [comment]
It should have been fixed in 0.19.something, feel free to reopen if the problem occurs again
